.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,17 @@
+---
+name: User story
+about: Suggest a feature from the perspective of a specific user role
+title: "[User story]"
+labels: ''
+assignees: ''
+
+---
+
+As a [user], 
+
+I want to [functionality], 
+
+so that [benefit].
+
+
+**Relates to issue:** #
```

Comment:
Do we need this or better to convert the other issue into this format? Or do you mean connecting this template to related issues to keep track of?

Reply:
I am actually not sure why I put that in there. I believe I took it from GitHub's feature request template.

Reply:
```suggestion
```